thunder/__init__.py

```diff
@@ -510,9 +510,13 @@ def get_computation_and_inputs(*args, **kwargs):
 
             prologue_traces = [prologue_trc]
             computation_traces = [computation_trc]
-            check_inplace_to_views(computation_trc)
+            orig_to_view_swap_map = check_inplace_to_views(computation_trc)
```

Comment:
I'm not totally happy about having this in `thunder/__init__.py`. Maybe we could have a function `check_and_functionalize_...` to which we 
 pass `compile_options.get("skip_inplace_functionalization", False)` ?